python/astshim/base.cc

```diff
@@ -27,6 +27,7 @@ namespace py = pybind11;
 using namespace pybind11::literals;
 
 namespace ast {
+namespace {
 
 PYBIND11_PLUGIN(base) {
     py::module mod("base", "Python wrapper for base.h");
```

Comment:
If you're modernizing the pybind11, why not [remove the uninformative docstrings](https://developer.lsst.io/coding/pybind11_style_guide.html#module-docstrings-should-be-empty)?